gamedb.yaml

```diff
@@ -4861,6 +4867,12 @@
   id: 218620
   compat_tool: proton-stable
 
+- name: "Persona 5 Strikers"
```

Comment:
Can you add a comment as to why GE Proton is needed?

Reply:
Added comment (Game doesn't start with standard proton 9.0-3)